tests/functional/testAuthenticationInApp.js

```diff
@@ -488,7 +488,7 @@ describe('User authentication', function() {
           queries.screenAuthenticationInApp.AUTH_START_BUTTON.visible()
               .enabled());
 
-      await vpn.waitForQuery(queries.screenAuthenticationInApp.AUTH_ERROR_POPUP_BUTTON.visible());
+      await vpn.waitForQuery(queries.screenAuthenticationInApp.AUTH_STUB_SET_PASSWORD_HEADLINE.visible());
```

Comment:
are there any opportunities to actually test this flow (ie get to this screen, create a password somehow, and click continue to sign in)?

Reply:
I managed to figure out a way to do this -- I had to modify the setup to get access to the underlying mock server, but it works well I think!